modelcontextprotocol/tools/assets.py

```diff
@@ -20,9 +21,10 @@ def update_assets(
         updatable_assets (Union[UpdatableAsset, List[UpdatableAsset]]): Asset(s) to update.
             Can be a single UpdatableAsset or a list of UpdatableAssets.
         attribute_name (UpdatableAttribute): Name of the attribute to update.
-            Only userDescription and certificateStatus are allowed.
+            Only userDescription,certificateStatus and readme are allowed.
         attribute_values (List[Union[str, CertificateStatus]]): List of values to set for the attribute.
             For certificateStatus, only VERIFIED, DRAFT, or DEPRECATED are allowed.
+            For readme, the value must be a valid HTML string without <html> and <body> tags OR a valid Markdown string.
```

Comment:
Check this

Reply:
Resolved via : https://github.com/atlanhq/agent-toolkit/pull/63/commits/4c78596e7cd2f739e532e1e25b482603b9bbf4a9